test/e2e/framework/deployment.go

```diff
@@ -28,6 +28,17 @@ func CreateDeployment(client kubernetes.Interface, deployment *appsv1.Deployment
 	})
 }
 
+// UpdateDeployment create Deployment.
+func UpdateDeployment(client kubernetes.Interface, deployment *appsv1.Deployment) {
+	ginkgo.By(fmt.Sprintf("Update Deployment(%s/%s)", deployment.Namespace, deployment.Name), func() {
+		gomega.Eventually(func() error {
+			_, err := client.AppsV1().Deployments(deployment.Namespace).Update(context.TODO(), deployment, metav1.UpdateOptions{})
+			gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
```

Comment:
```suggestion
```

You can't use `gomega.Expect` in `Eventually`. Otherwise it will fail the test immediately.

Reply:
If it's a conflict error, you shall refresh it before update.